src/app/pages/fund/components/fund-account-tile.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, Stack, color, transition } from '@stacks/ui';
+import { Box, Stack, color } from '@stacks/ui';
```

Comment:
Maybe we can swap these components out too? I don't want it to be a never-ending task either though